src/js/components/DateInput/utils.js

```diff
@@ -126,20 +135,26 @@ export const textToValue = (text, schema, valueProp) => {
       parts.d > 31
     )
       return parts;
+    // parts look legit
```

Comment:
not completely clear what this comment means

Reply:
Updated the comments in this area.